leapp/messaging/answerstore.py

```diff
@@ -50,15 +51,33 @@ def update(self, answer_file, allow_missing=False):
             conf.write(afile)
         return not_updated
 
+
+    def _load_ini(self, inifile):
+        """
+        Loads an ini config file from the given location.
+
+        :param inifile: Path to the answer file to load.
+        :return: configparser.SafeConfigParser object
+        :raises CommandError if any of the values are not in key=value format
+        """
+        conf = configparser.SafeConfigParser(allow_no_value=False)
+        try:
+            conf.read(inifile)
+            return conf
+        except configparser.ParsingError as exc:
+            # Some of the sections were not in key = value format
+            raise CommandError(exc.message)
```

Comment:
@fernflower @vinzenz   shouldn't be this rather `ValueError`? Or something else? Just curious why `CommandError`

Reply:
My first thought was to take some leapp specific exception as we are dealing with leapp answerfile but I have no hard feelings about it.

Reply:
I think CommandError is fine, because it will be caught by leapp and displayed as an appropriate failure of the execution. However I feel like it's necessary to bring a bit more light here. 

In the end I'd prefer if it would read like this:
```
Failed to load answer file `{inifile}` with following errors: {exc.message}
```

Reply:
I'd like to see a bit more of a message like this:
```
Failed to load answerfile `{inifile}` due to errors reading the file: {exc.message}
```